src/ChartBlocks/Client.php

```diff
@@ -91,7 +103,8 @@ public function __get($name) {
      * 
      * @return string
      */
-    public function getApiUrl() {
+    public function getApiUrl(): string
```

Comment:
There's quite a few changes in this PR that are the result of hitting the "Cleanup code" button in PHPStorm which basically amounts to adding a return type signature to functions. Just a heads up.